### PR TITLE
Transfer page and fs listener bugfixes

### PR DESCRIPTION
**Pull request by bpar476**
* Fix some null dereferences in transfer page
* Drop thumbnailable filter from issue listener - it was causing NPE's and missing the thumbnails

### CODE REVIEW

**Comment by bpar476** [comment]
Created this Bug to track the `thumbNailable` API issue https://aws-partner.atlassian.net/browse/CHET-414

File: core/src/main/java/com/atlassian/migration/datacenter/core/fs/listener/JiraIssueAttachmentListener.java [hunk 1]
```diff
@@ -66,8 +67,10 @@ public void onIssueEvent(IssueEvent issueEvent) {
 
             attachments
                     .stream()
-                    .filter(Attachment::isThumbnailable)
+                    // FIXME: I lie about thumbnailable
+                    //.filter(Attachment::isThumbnailable)
                     .map(attachmentStore::getThumbnailFile)
+                    .filter(Objects::nonNull)
                     .map(File::toPath)
```
Comment (theghostwhoforks) [inline core/src/main/java/com/atlassian/migration/datacenter/core/fs/listener/JiraIssueAttachmentListener.java:74]: Do we need to add a filter(Files::exists) here in the chain? i.e. the case when DB says there's a thumbnail, but the FS does not contain the file?

Comment (bpar476) [reply core/src/main/java/com/atlassian/migration/datacenter/core/fs/listener/JiraIssueAttachmentListener.java:74]: That's a good idea. I'll create a bug for it :)

Comment (bpar476) [reply core/src/main/java/com/atlassian/migration/datacenter/core/fs/listener/JiraIssueAttachmentListener.java:74]: https://aws-partner.atlassian.net/browse/CHET-415

**Pull request merged**